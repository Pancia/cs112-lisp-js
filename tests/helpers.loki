(print 5)
(print "print")
(print (get [0 1 2] 1))
(print (get {"x" 5} "x"))
(print (assoc {} "x" "y"))
(print (range 5))
(print (+ 1 2 3))
(print (- 1 2 3))
(print (* 1 2 3))
(print (/ 1 2 3))
(print (mod 1 2))
(print (and true false))
(print (or true false))
(print (= 5 5))
(print (= 5 "5"))
(print (!= 5 3))
(print (!= 3 "3"))
(print (< 1 3 5 7))
(print (<= 1 3 3 7))
(print (> 5 4 3))
(print (>= 6 6 6 0))
