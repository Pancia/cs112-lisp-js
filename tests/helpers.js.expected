loki.print(5)
loki.plus(1, 2, 3)
